tests/platforms/test_oscn.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import call, patch
```

Comment:
Do these tests run against the live site? If so, we should disable or delete until we've integrated use of fixtures and mocks to produce deterministic tests.